package.json

```diff
@@ -11,13 +11,12 @@
     "yarn": "1.22.0"
   },
   "scripts": {
-    "postinstall": "cd server && yarn install && cd ../client && yarn install",
-    "start:server": "cd server/ && yarn start",
-    "start:client": "cd client/ && yarn start",
-    "build:server": "cd server/ && yarn build",
-    "build:client": "cd client/ && yarn build",
-    "move:client": "mv ./client/build ./server/website",
-    "prepare:client": "yarn build:client && yarn move:client",
+    "start:server": "cd server && yarn start",
+    "start:client": "cd client && yarn start",
+    "build:server": "cd server && yarn build",
+    "build:client": "cd client && yarn build",
+    "build:clientGeneric": "cd client/ && yarn build:generic",
```

Comment:
I wasn't sure about the name here. The fact that we do also production build for the local machine as we need service workers prevent me from calling it production one